services/centralized-kubecost/2.5.0/cosi-storage/cosi-bucket.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta2
+kind: HelmRelease
+metadata:
+  name: kubecost-cosi-storage
+  namespace: ${releaseNamespace}
+spec:
+  chart:
+    spec:
+      chart: cosi-bucket-kit
+      sourceRef:
+        kind: HelmRepository
+        name: takirala
```

Comment:
revert?

Reply:
right, was waiting for https://github.com/mesosphere/charts/pull/1534 to merged. now that its merged, i dropped this custom helm repo reference.